pr_testing/run-pr-hlt-p2-integration.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -ex
+source $WORKSPACE/cms-bot/pr_testing/setup-pr-test-env.sh
+
+PR_REPO_NUM=$(echo $PULL_REQUEST | sed 's|^.*/||;s|#||')
+UPLOAD_PATH="${CMSSW_VERSION}-${PR_REPO_NUM}/${ARCHITECTURE}/${BUILD_NUMBER}"
+# Report test started
+mark_commit_status_all_prs 'hlt-p2-integration' 'pending' -u "${BUILD_URL}" -d "Running"
+
+ls -l /eos/cms/store
+# Do work
+HLT_P2_SCRIPT="src/HLTrigger/Configuration/scripts"
+HLT_BASEDIR="${CMSSW_BASE}"
+if [ ! -e "${HLT_BASEDIR}/${HLT_P2_SCRIPT}" ] ; then HLT_BASEDIR="${CMSSW_RELEASE_BASE}" ; fi
+cp -r ${HLT_BASEDIR}/${HLT_P2_SCRIPT} $WORKSPACE/rundir
+rm -rf $WORKSPACE/rundir/__pycache__
+
+pushd $WORKSPACE/rundir
+  export LOCALRT=${WORKSPACE}/${CMSSW_VERSION}
+  timeout $TIMEOUT ${HLT_BASEDIR}/${HLT_P2_SCRIPT}/hltPhase2UpgradeIntegrationTests 2>&1 | tee -a ${WORKSPACE}/hlt-p2-integration.log
+popd
+
+# Upload results
+source $WORKSPACE/cms-bot/jenkins-artifacts
+touch ${RESULTS_DIR}/15-hlt-p2-integration-report.res ${RESULTS_DIR}/15-hlt-p2-integration-failed.res
+if [ -f $WORKSPACE/rundir/Phase2Timing_resources.json ] ; then
```

Comment:
@mmusich , so `hltPhase2UpgradeIntegrationTests` is support to generate this json file?